docs/forms-apps.md

```diff
@@ -13,10 +13,12 @@
 
 ## Constructor
 
-| Parameter           | Required | Type     | Description                      |
-| ------------------- | -------- | -------- | -------------------------------- |
-| `options.accessKey` | Yes      | `string` | Access key provided by OneBlink. |
-| `options.secretKey` | Yes      | `string` | Secret key provided by OneBlink. |
+| Parameter                   | Required | Type     | Description                                                                  |
+| --------------------------- | -------- | -------- | ---------------------------------------------------------------------------- |
+| `options.accessKey`         | Yes      | `string` | Access key provided by OneBlink.                                             |
+| `options.secretKey`         | Yes      | `string` | Secret key provided by OneBlink.                                             |
+| `options.regionCode`        | No       | `string` | Sets the default apiOrigin to the region appropriate value. Defaults to `AU` |
+| `options.oneBLinkAPIOrigin` | No       | `string` | Overrides the apiOrigin set by default or regionCode.                        |
```

Comment:
```suggestion
| `options.oneBlinkAPIOrigin` | No       | `string` | Overrides the apiOrigin set by default or regionCode.                        |
```

Reply:
Actually, can we remove this from here as it is already in the `development.md` doc. Could we update that doc to outline that all public classes have the `oneBlinkAPIOrigin` option in the constructor.

Reply:
You want all that line in all the class docs to be gone? And replace with notifier in dev doc?

Reply:
Yep yep :)